src/Libraries/PythonNodeModelsWpf/IronPythonCompletionProvider.cs

```diff
@@ -47,6 +44,38 @@ public ScriptScope Scope
             set { scope = value; }
         }
 
+        /// <summary>
+        /// Cache storing a reference to the Dynamo Core directory
+        /// </summary>
+        private static string dynamoCoreDir { get; set; }
```

Comment:
I think we can remove this property, see comments below

Reply:
@alfarok I agree with @QilongTang - that seems like a good place to pass the references you need.